docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.md

```diff
@@ -78,6 +79,42 @@ In compose mode you must call the [`saveAsync`](Office.context.mailbox.item.md#s
 ---
 ---
 
+#### masterCategories :[MasterCategories](/javascript/api/outlook/office.mastercategories)
+
+Gets an object that provides methods to manage the categories master list on this mailbox.
```

Comment:
Consider "Gets an object that provides methods to manage" -> "Gets an object to manage".